src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,51 @@
+package codeu.controller;
```

Comment:
Please add tests for AdminServlet controller

Reply:
Done, I hard coded the users who can access it